test/distributed/fsdp/test_fsdp_ignored_modules.py

```diff
@@ -173,29 +165,32 @@ def _test_ignored_modules_transformer(
             CUDAInitMode.CUDA_BEFORE,
             deterministic=True,
         )
+        if use_auto_wrap:
+            nonwrapped_model.output_proj.weight = nn.Parameter(
```

Comment:
why do we need this reassignment?

Reply:
See the comment above about unsharing the weight.

Reply:
I can duplicate the comment here if it helps.